allennlp_models/rc/dataset_readers/transformer_squad.py

```diff
@@ -50,11 +66,19 @@ class TransformerSquadReader(DatasetReader):
         is called "stride" instead of "overlap" because that's what it's called in the original huggingface
         implementation.
     skip_invalid_examples: `bool`, optional (default=`False`)
-        If this is true, we will skip examples that don't have a gold answer. You should set this to True during
-        training, and False any other time.
+        If this is true, we will skip examples where the question+context is truncated according to `length_limit`
+        such that the context no longer contains a gold answer. For SQuAD v1.1-style datasets, you should set this
+        to `True` during training, and `False` any other time.
     max_query_length : `int`, optional (default=`64`)
         The maximum number of wordpieces dedicated to the question. If the question is longer than this, it will be
         truncated.
+    use_cls_token_for_unanswerable : `bool`, optional (default=`False`)
+        If this is true, we'll use the `[CLS]` token as the gold answer for questions that don't have a gold
+        answer. For SQuAD v2.0-style datasets that include impossible questions, this should
+        be set to `True`.
+    cls_token : `str`, optional (default=`[CLS]`)
+        The `[CLS]` token of the tokenizer. If `use_cls_token_for_unanswerable` is `False`, you don't
+        need to worry about this.
```

Comment:
Can't we get this from the tokenizer? Why do we have to specify it here?

Reply:
I don't think there's a way to get this from the tokenizer 100% of the time, because some tokenizers, like RoBERTa, don't have a `[CLS]` token, right?

Reply:
It's just called `<s>` in RoBERTa.